### PR TITLE
Document the need to have fully qualified names for global plugin settings

### PR DESCRIPTION
**Pull request by fanf**
See https://github.com/sbt/sbt/issues/2364

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @fanf,

Thank you for your contribution! We really value the time you've taken to put this together.

We see that you have signed the Typesafe Contributors License Agreement before, however, the CLA has changed since you last signed it.
Please review the new CLA and sign it before we proceed with reviewing this pull request:

[http://www.typesafe.com/contribute/cla](http://www.typesafe.com/contribute/cla)

File: src/reference/02-DetailTopics/02-Configuration/05-Global-Settings.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ shellPrompt := { state =>
 }
 ```
 
+You can also configure plugins globaly added in `$global_plugin_sbt_file$`
```
Comment (SethTisue) [inline src/reference/02-DetailTopics/02-Configuration/05-Global-Settings.md]: globally

File: src/reference/02-DetailTopics/02-Configuration/05-Global-Settings.md [hunk 2]
```diff
@@ -21,6 +21,17 @@ shellPrompt := { state =>
...
+(see next paragraph) in that file, but you need to use fully qualified
+name for their properties. For example, for sbt-eclipse property `withSource`
```
Comment (SethTisue) [inline src/reference/02-DetailTopics/02-Configuration/05-Global-Settings.md]: s/name/names/

**Comment by eed3si9n** [comment]
It's not just the global settings, but for `*.scala` files in general, so perhaps we should add full explanation in http://www.scala-sbt.org/0.13/tutorial/Organizing-Build.html in the getting started and link to it from Global Settings page.

**Comment by fanf** [comment]
@SethTisue what is the convention here? Do you prefer that I make new commits with the correction, or a rebase ? Or commits until just before the merge (and here a rebase) ? For now, I'm going to add a commit, waiting for feedbacks.

**Comment by SethTisue** [comment]
I'm just a spectator here, I'll let Eugene answer

**Comment by eed3si9n** [comment]
I am not too fussy about commit history on sbt/website. You can keep adding commits.

**Pull request merged**